Game/Source/GDKTestGyms/Private/CounterComponent.cpp

```diff
@@ -21,32 +21,53 @@ void UCounterComponent::TickComponent(float DeltaTime, ELevelTick TickType, FAct
 	if (Timer <= 0.0f)
 	{
 		Timer = ReportFrequency;
-		UpdateCachedActorCounts();
+		UpdateCachedAuthActorCounts();
 	}
 }
 
-int32 UCounterComponent::GetActorClassCount(TSubclassOf<AActor> ActorClass) const
+int32 UCounterComponent::GetActorTotalCount(const TSubclassOf<AActor>& ActorClass) const
```

Comment:
Seems like this function is not used anywhere

Reply:
This is true! The counter component now counts the number of actors a worker has interest in as well as the number of actors it has auth over. Currently we don't actually use the interest actor count anywhere...

I left this in as it seems like it could potentially be useful and is very lightweight. Mike also liked the idea of keeping this code around.

You are right though this could be removed... I am in two minds about it...